kafka/src/main/java/org/apache/apex/malhar/kafka/AbstractKafkaInputOperator.java

```diff
@@ -366,17 +374,19 @@ public String getClusters()
     return Joiner.on(';').join(clusters);
   }
 
-  public void setTopics(String... topics)
+  public void setTopics(String topics)
   {
-    this.topics = topics;
+    Topic.validate(topics.replace(" |,", "");
+    this.topics = Iterables.toArray(Splitter.on(',').trimResults().omitEmptyStrings().split(topics), String.class);
```

Comment:
This assumes that " " is also a valid separator for topics. I think it is better to validate each topic separately.